public/js/operation.js

```diff
@@ -120,26 +121,27 @@ function submitTx() {
 }
 
 function openIdenaApp() {
-    toastr.success("Openning Idena app");
+    toastr.info("Openning Idena app");
     let address = global_variables.IDENA_WALLET;
     let amount = window.amount;
     let bscAddress = window.address;
-    let url = `dna://send/v1?address=${address}&amount=${amount}&comment=BSCADDRESS${bscAddress}`
+    let callback_url = global_variables.BRIDGE_URL + "/swaps/assign?uuid=" + window.uuid + "&redirect=true";
```

Comment:
I guess _"/submit?uuid="_ must be here instead of _"/swaps/assign?uuid="_

Reply:
Yes that's true , i didn't test the last commit before adding to the PR , I will edit  it